modules/hostap/Kconfig

```diff
@@ -25,7 +25,7 @@ if WIFI_NM_WPA_SUPPLICANT
 
 config HEAP_MEM_POOL_ADD_SIZE_HOSTAP
 	def_int 66560 if WIFI_NM_HOSTAPD_AP
-	def_int 48000 if WIFI_NM_WPA_SUPPLICANT_CRYPTO_ENTERPRISE
+	def_int 55000 if WIFI_NM_WPA_SUPPLICANT_CRYPTO_ENTERPRISE
```

Comment:
is this change only for Wi-Fi cred + enterprise? If this is independent of Wi-Fi cred please move to a different commit with reason for the increase. If its only for Wi-Fi cred + enterprise then add a new conditional.

Reply:
ok